### ISSUE TITLE
Chef-3694 warnings in recipes/etcd_cluster.rb and recipes/master_cluster.rb

### ISSUE THREAD

**Issue by jperville** [IshentRas/cookbook-openshift3#121]
When bootstrapping a cluster, I get a lot of CHEF-3694 spam related to duplicated resources between recipe/etcd_cluster.rb and recipes/master_cluster.rb

Compare:
 - https://github.com/IshentRas/cookbook-openshift3/blob/master/recipes/etcd_cluster.rb#L16-L24 vs https://github.com/IshentRas/cookbook-openshift3/blob/master/recipes/master_cluster.rb#L30-L38
 - https://github.com/IshentRas/cookbook-openshift3/blob/master/recipes/etcd_cluster.rb#L26-L28 vs https://github.com/IshentRas/cookbook-openshift3/blob/master/recipes/master_cluster.rb#L26-L28
 - https://github.com/IshentRas/cookbook-openshift3/blob/master/recipes/etcd_cluster.rb#L30-L34 vs https://github.com/IshentRas/cookbook-openshift3/blob/master/recipes/master_cluster.rb#L40-L44
 - https://github.com/IshentRas/cookbook-openshift3/blob/master/recipes/etcd_cluster.rb#L36-L39 vs https://github.com/IshentRas/cookbook-openshift3/blob/master/recipes/master_cluster.rb#L40-L44

In other words, the code is duplicated in two places: https://github.com/IshentRas/cookbook-openshift3/blob/master/recipes/etcd_cluster.rb#L16-L39 and https://github.com/IshentRas/cookbook-openshift3/blob/master/recipes/master_cluster.rb#L26-L49 .

If you have time @IshentRas can you do a PR? Else I will look it up when I have some free time.

**Comment by jperville** [comment]
Should be fixed in #133
